tests/jax/common.libsonnet

```diff
@@ -205,4 +205,18 @@ local tpus = import 'templates/tpus.libsonnet';
       ||| else '',
```

Comment:
Not your change, but what's up with maybeInstallLibtpuV4 above? Can this be removed?